src/main/java/uk/gov/hmcts/reform/em/orchestrator/service/orchestratorcallbackhandler/CcdCallbackBundleUpdater.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.em.orchestrator.service.orchestratorcallbackhandler;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.em.orchestrator.service.ccdcallbackhandler.CcdCallbackDto;
+import uk.gov.hmcts.reform.em.orchestrator.service.dto.CcdBundleDTO;
+import uk.gov.hmcts.reform.em.orchestrator.service.dto.CcdDocument;
+
+import java.io.IOException;
+
+@Service
+class CcdCallbackBundleUpdater {
+
+    private final Logger log = LoggerFactory.getLogger(CcdCallbackBundleUpdater.class);
+
+    private final ObjectMapper objectMapper;
+
+    CcdCallbackBundleUpdater(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
+
+    void updateBundle(CcdCallbackDto ccdCallbackDto, StitchingCompleteCallbackDto stitchingCompleteCallbackDto)
+            throws CallbackException, IOException {
+        ArrayNode bundles = ccdCallbackDto
+                .findCaseProperty(ArrayNode.class)
+                .orElseThrow(() -> new CallbackException(400, null, "Bundle collection could not be found"));
```

Comment:
At some point we should think about extracting this code into a bundle extractor, we've got similar code in other places I think

Reply:
we can refactor at some point